UI/src/app/file-upload/file-upload.component.ts

```diff
@@ -58,7 +58,7 @@ export class FileUploadComponent implements OnInit, OnDestroy {
             this.dataList.push(new Dataset(id, fileName));
             alert(response.fileName + " has been uploaded to database!");
         }else{
-            alert(response.Error);
+            alert("Upload Failed! Please reset the database! See the Readme file.");
```

Comment:
or upload a file with adifferent name

Reply:
should not try with different filename.. bcoz there are few things hardcoded for that particular kaggle dataset name.